ProcessMaker/Events/TokenCreated.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace ProcessMaker\Events;
+
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Broadcasting\PresenceChannel;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Auth;
+use Laravel\Passport\Token;
+use ProcessMaker\Contracts\SecurityLogEventInterface;
+use ProcessMaker\Models\User;
+
+class TokenCreated implements SecurityLogEventInterface
+{
+    use Dispatchable, InteractsWithSockets, SerializesModels;
+
+    public $user;
+    public $data;
+
+    /**
+     * Create a new event instance.
+     *
+     * @return void
+     */
+    public function __construct(Token $created_values)
+    {
+        $this->user = Auth::user();
```

Comment:
SecurityLogger already get the session's userid, so `$this->user` is not needed here, as for these events the log record will not be sent to a job queue.